proglogica/logica_teoria/aplicaciones.html

```diff
@@ -332,6 +334,66 @@ <h2 class="title"><strong>Consultas lógicas basadas en reglas</strong></h2>
             </div>
         </section><!-- #content -->
         
+        <!--==========================
+        Content Section
+        ============================-->
+        <section id="Logica_probabilistica" style="margin-top:50px;">
+            <div class="container">
+                <div class="text_cell_render border-box-sizing rendered_html">
+                    <h2 class="title"><strong>Programación Lógica Probabilística</strong></h2>
+                    <p> La programación probabilística es un paradigma de programación en el que se especifican modelos 
+                        probabilísticos y la inferencia a partir de estos modelos se realizan automáticamente. 
+                        Además se ha utilizado como un subcampo de la inteligencia artificial, el cual incluye a 
+                        los lenguajes de programación tradicionales estructuras para trabajar con inferencia probabilística 
```

Comment:
```suggestion
                        los lenguajes de programación tradicionales estructurados para trabajar con inferencia probabilística 
```
O modificarlo de acuerdo con lo que se haya querido decir.